bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -1093,6 +1100,10 @@ journalFormatVersionToWrite, getBufferedChannelBuilder(),
                                     numEntriesToFlush--;
                                     entry.run();
                                 }
+
+                                if (forceWriteThread.requestProcessor != null) {
+                                    forceWriteThread.requestProcessor.flushPendingResponses();
+                                }
```

Comment:
Do we need to record [`journalStats.getJournalFlushStats()`](https://github.com/apache/bookkeeper/pull/3837/files#diff-0a79d54e3dfd4bab1d85510ef957f8f4b7ab6f6acd09aadd634b4d7bbee6e1f4R1111) before trigger flushPendingResponses? That would lead the `JournalFlushStats` to include response time.

Reply:
The flush would be very fast though. It's only posting an event on the channel group, not writing the actuals responses on the channels.